src/ray/object_manager/plasma/store.cc

```diff
@@ -614,6 +624,8 @@ void PlasmaStore::SealObjects(const std::vector<ObjectID> &object_ids) {
     object_info.owner_worker_id = entry->owner_worker_id.Binary();
     object_info.metadata_size = entry->metadata_size;
     infos.push_back(object_info);
+
+    num_bytes_unsealed_ -= entry->data_size + entry->metadata_size;
```

Comment:
Is it possible to add a unit test that we are tracking this quantity correctly?

Reply:
Hmm the plasma store doesn't currently have unit tests, but let me try...

Reply:
Uh yeah, this is turning out to be pretty annoying since we need to mock out the plasma protocol and allocator. Okay if we do this later?

Reply:
Yeah sounds good. Perhaps we can at least RAY_CHECK the following variant:

if the number of unsealed objects is zero, RAY_CHECK num_bytes_unsealed_ == 0.